subcommand/consul-sidecar/command.go

```diff
@@ -44,6 +54,11 @@ func (c *Command) init() {
 		"Log verbosity level. Supported values (in order of detail) are \"trace\", "+
 			"\"debug\", \"info\", \"warn\", and \"error\". Defaults to info.")
 
+	// TODO: do these need to be validated if they're validated before being passed in?
```

Comment:
QUESTION: Do we need to validate these flags again? They are being validated by inject-connect, and then values that are passed to consul-sidecar are done via code.